tinyquery/evaluator_test.py

```diff
@@ -945,6 +945,32 @@ def test_null_count_distinct(self):
             'SELECT COUNT(DISTINCT val1) FROM some_nulls_table',
             self.make_context([('f0_', tq_types.INT, [2])]))
 
+    def test_group_concat(self):
+        self.assert_query_result(
+            'SELECT GROUP_CONCAT(str) FROM string_table',
+            self.make_context([
+                ('f0_', tq_types.STRING, ['hello,world'])
+            ]))
+        self.assert_query_result(
```

Comment:
Is this the case that's testing a repeated field?

[ I apologize for the silly quesstion; I'm not actually that familiar with tinyquery, but the author has left Khan so I'm the best that's left. :-) ]